benchmark/python/benchmark_e2e.py

```diff
@@ -0,0 +1,184 @@
+# This is an end-to-end benchmarking script for any ONNX model.
+#
+# Prerequisites: 
+# 0) Install onnxruntime-genai and onnxruntime
+#
+# 1) Use builder.py to build the desired ONNX model
+#
+# 2) Run this script with the desired arguments. Run benchmark_e2e.py -h for help.
+
+import onnxruntime_genai as og
+import time
+import argparse
+
+# Use input model to generate prompt
+def generate_prompt(model, tokenizer, prompt_length) -> str:
+    prompt = "a"
```

Comment:
Should we use an actual small prompt (e.g. "The quick brown fox")?

Reply:
Might be a different amount of tokens depending on tokenizer but may not be a problem... Output from this single token prompt looks good usually though